src/ontology/mondo-edit.obo

```diff
@@ -534920,14 +534920,20 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 id: MONDO:0033664
 name: Kilquist syndrome
 def: "An autosomal recessive multisystem disorder characterized by neurologic, gastrointestinal, and secretory dysfunction. Affected individuals present at birth with hypotonia, feeding difficulties, mild dysmorphic features, and sensorineural hearing loss. They show poor overall growth associated with gastrointestinal anomalies such as gastroesophageal reflux or midgut malrotation, as well as profound global developmental delay with inability to sit or speak. Tear, sweat, and saliva production is also impaired, causing dry mouth and recurrent bronchial mucus plugging. Some of the clinical features are reminiscent of cystic fibrosis." [OMIM:619080, PMID:30740830, PMID:32754646]
+subset: gard_rare {source="MONDO:GARD"}
+subset: ordo_subtype_of_a_disorder {source="Orphanet:633021"}
 subset: otar {source="MONDO:OTAR"}
+subset: rare
 synonym: "KILQS" EXACT ABBREVIATION [OMIM:619080]
+synonym: "SLC12A2-related autosomal recessive neonatal-developmental delay-intellectual disability-feeding difficulty-sensorineural deafness syndrome" EXACT [https://orcid.org/0000-0001-5493-2602, OMIM:619080]
 xref: MEDGEN:1742639 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: OMIM:619080 {source="MONDO:equivalentTo"}
+xref: Orphanet:633021 {source="MONDO:mondoIsNarrowerThanSource"}
```

Comment:
@MeeSiing I would expect this x-ref to be equivalent. 
If it is not equivalent, then the merge should not have happened (because it means that these 2 terms were not equivalent). 
Could you give me some insight into why you make this orphanet ID "mondoisnarrowerthansource"?